src/database/migrations/V1__addTest.ts

```diff
@@ -7,6 +7,7 @@ export async function up(db: knex) {
   await db.schema.createTable(TEST_TABLE, (table) => {
     table.uuid('id').primary();
     table.uuid('user_id').notNullable();
+    table.uuid('administering_user_id').notNullable();
```

Comment:
In the test results table, we have a column performing a similar task, but it's called `created_id`, just wondering if they should share a more similar name?

Reply:
Good point. Maybe we should unify all of this as:

```
creator_user_id: ``
creator_confidence: ``
```

Then on the API layer we can rename `creation` to `administration` for `test` and `entry` for `result`

Reply:
What about if they're both:
```
creator: {
  user_id: ...,
  confidence: ...
}
```
Feels a bit weird to have the same content under different keys?

Reply:
Yeah.. this is true... let's go with this as it is simpler.. I'll change accrodingly